Server/src/models/interfaces/DatasetModelInterface.ts

```diff
@@ -73,3 +73,8 @@ export interface IApprovalDatasetModel extends IClientDatasetModel {
     datasetIsFlagged: number
     datasetFlaggedComment: string
 }
+
+export interface IUserDatasets {
+    datasetID: number

```

Comment:
no need to capitalize 'D' in the ID

Reply:
It was requested to be capitalized in the story, I will change